### ISSUE TITLE
Healing Ruby II stripping off Sneak/Invisible from party.

### ISSUE THREAD

**Issue by joebinne** [EdenServer/community#2283]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [X] I have provided my client version in the details. (type /ver into your game window)

### Details

Basically, If you have sneak/invisible on yourself or a nearby party members and use carby's Healing Ruby II, it well insta-remove both buffs. It doesn't seem to affect barspells and/or reraise. 

Steps:
1. Be in a pt.
2. Have a SMN/WHM sneak/inv someone.
3. Use Healing Ruby II.

/ver is of Eden v4.3

**Comment by panfunk** [comment]
#1311

**Comment by rude-jerk** [comment]
Resolved in patch on December 21, 2020.
